load/UpdateDBPLanguageTranslation.py

```diff
@@ -56,7 +67,7 @@ def _updateOrInsertLanguageTranslations(self, languageNames, languageSourceId, l
                 insertRows = []
                 updateRows = []
 
-                indexedKey = "%s%s%s" % (languageSourceId,languageName.lower(),priority)
+                indexedKey = "%s%s%s%s" % (languageSourceId, languageTranslationId, languageName.lower(), priority)
```

Comment:
this indexedKey is incorrect when priority = 9. So I think we should pass in the indexed key

this query statement will change from an insert to an update when the code is correct
INSERT INTO language_translations (name, priority, language_source_id, language_translation_id) VALUES ('Ishkashimi', '9', '2580', '2580'); -- should be update, old value Eshkashimi

Reply:
@bradflood , I have updated the log file.

https://github.com/faithcomesbyhearing/dbp-etl/files/10078908/log_update_languages_translation.log

Also, you can see in the following line that the above language would be updated.

```sql
UPDATE language_translations SET name='Ishkashimi' WHERE language_source_id='2580' AND language_translation_id='2580' AND priority='9' AND id='134823';

```